### DISCUSSION TITLE
Easy Font Selection

### DISCUSSION THREAD

**Discussion by quakec** [WerWolv/ImHex#1495] [Feedback]
In settings, it would be nice to have a dropdown of some preset font types and sizes (namely the latest default UI font per O/S, for windows 11 it was "Segoe UI Variable", set to size 18 in my case, I felt it was a little far-fetched having to copy the font out of the c:\windows\font folder to then point to the file through the open dialog in settings, just to have something that made the UI look like it was in its natural habitat :)

**Comment by WerWolv** [comment]
Hey! This and more has been implemented since :)

<img width="371" alt="image" src="https://github.com/user-attachments/assets/51a5b432-523c-48ea-ac86-7804f7e2b4c4" />
